src/components/contact-info/ContactInfo.vue

```diff
@@ -3,83 +3,130 @@
     <!-- SUMMARY SECTION -->
     <template v-if="!isEditing">
       <!-- Display Contact Info -->
-      <v-row no-gutters class="edit-section">
-        <v-col cols="12" sm="3" class="pr-4">
+      <v-row
+        no-gutters
+        class="edit-section"
+      >
+        <v-col
+          cols="12"
+          sm="3"
+          class="pr-4"
+        >
           <label class="contact-info-label">
             <span>{{ contactLabel }} Contact Information</span>
-            <v-flex>
-              <v-chip v-if="hasBusinessContactInfoChange" x-small label color="primary" text-color="white">
-                {{editedLabel}}
+            <v-col>
```

Comment:
I want to test this, and maybe replace it with d-flex.

Reply:
As above, I replaced this with a div.

![image](https://github.com/bcgov/bcrs-shared-components/assets/10002889/a0648e0a-1274-4417-8f0a-ce3f947fadd9)